blocks/autocompleters/index.js

```diff
@@ -137,5 +137,6 @@ export function userAutocompleter() {
 		getOptions,
 		allowNode,
 		onSelect,
+		isDebouncedCompleter: true,
```

Comment:
Since this is a property of a completer object, it seems redundant to include "completer" in the property name.

Reply:
good suggestion, addressed in https://github.com/WordPress/gutenberg/pull/4406/commits/b9fd29afc6825992b336921f83e77086ded65ac5